cdap-docs/hydrator-manual/source/developing-plugins/creating-a-plugin.rst

```diff
@@ -172,7 +195,10 @@ Example::
 
 Batch Sink Plugin
 =================
-In order to implement a Batch Sink (to be used with the *Data Pipeline* artifact), you extend the
+A ``BatchSink`` plugin is used to write data in both batch and realtime data pipelines.
+It is used to prepare and configure the output of a batch of data.
```

Comment:
didn't want to use 'pipeline run', because the same run of a streaming pipeline will write many batches.